.github/workflows/phpunit-tests.yml

```diff
@@ -154,13 +154,45 @@ jobs:
       phpunit-config: ${{ matrix.multisite && 'tests/phpunit/multisite.xml' || 'phpunit.xml.dist' }}
       report: ${{ matrix.report || false }}
 
+  #
+  # Runs specific test groups.
+  #
+  specific-test-groups:
+    name: PHP ${{ matrix.php }}
```

Comment:
You really have to know what you're looking for to find these tests, they're under:

> PHP 8.3 / PHP 8.3 / MySQL 8.0 (html-api-html5lib-tests)


<kbd>![Screenshot 2024-11-08 at 12 31 00](https://github.com/user-attachments/assets/f27022e6-5332-4c2b-accb-db8897b1d4c2)</kbd>

For specific tests like this, does it make sense to bring the groups to a higher level? The group seems most relevant at least in this case.

Reply:
I've gone and made some adjustments to how the jobs are named so that each test group will have its own top level group with PHP/DB combinations listed under it.